relay-event-schema/src/protocol/contexts/mod.rs

```diff
@@ -90,6 +92,9 @@ pub enum Context {
     Nel(Box<NelContext>),
     /// Performance score information.
     PerformanceScore(Box<PerformanceScoreContext>),
+    /// Ourlogs (logs product) information.
+    #[metastructure(tag = "sentry_logs")]
+    OurLogs(Box<OurLogsContext>),
```

Comment:
Are these hard to remove later or is the deprecation sticky? If they are I'm probably going to delay this a week until Abhi is back and we can make sure everyone is fine with the context on the sdk side.

Reply:
Removing the variant from the code is not a problem. After removing it, relay will still forward the context as if it were a user-defined context.

Reply:
Great